client/my-sites/plugins/plugin-activate-toggle/test/index.jsx

```diff
@@ -1,83 +1,67 @@
 /**
  * External dependencies
  */
-var chai = require( 'chai' ),
-	expect = chai.expect,
-	ReactDom = require( 'react-dom' ),
-	React = require( 'react' ),
-	mockery = require( 'mockery' ),
-	sinon = require( 'sinon' ),
-	noop = require( 'lodash/noop' ),
-	TestUtils = require( 'react-addons-test-utils' );
+import { expect } from 'chai';
+import identity from 'lodash/identity';
+import mockery from 'mockery';
+import { mount } from 'enzyme';
+import React from 'react';
+import { spy } from 'sinon';
 
 /**
- * Mocks & fixtures
+ * Internal dependencies
  */
-var mockedPluginAction = require( './mocks/plugin-action' ),
-	fixtures = require( './fixtures' ),
-	mockedActions = require( './mocks/actions' );
-
-require( 'lib/react-test-env-setup' )();
+import EmptyComponent from 'test/helpers/react/empty-component';
+import fixtures from './fixtures';
+import mockedActions from './mocks/actions';
+import mockedPluginAction from './mocks/plugin-action';
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
 
 describe( 'PluginActivateToggle', function() {
-	var PluginActivateToggle, analyticsMock;
-
-	analyticsMock = {
-		ga: { recordEvent: sinon.spy() },
-		tracks: { recordEvent: sinon.spy() }
+	const analyticsMock = {
+		ga: { recordEvent: spy() },
+		tracks: { recordEvent: spy() }
 	};
+	let PluginActivateToggle;
+
+	useFakeDom();
+	useMockery();
```

Comment:
Surprised you didn't need to import `mount` separately in a `before` block as you had to in the other test.